controls/radrichtextbox/features/selection.md

```diff
@@ -10,32 +10,112 @@ published: True
 # Selection
 
 
-
-The __RadRichTextBox__ supports not only selection via the UI, but also programmatic selection. This topic covers the following:
+**RadRichTextBox** supports not only selection via the UI, but also programmatic selection. This topic covers the following:
 
 * [Use UI Selection](#ui-selection)

```

Comment:
You don't need those lists anymore. The new documentation design will automatically add all h2 to h(something don't remember the number) to an aside menu that floats and you can select a section from there.